helpers/test/test_lib_tasks_docker_release.py

```diff
@@ -182,3 +179,250 @@ def test_docker_build_multi_arch(self) -> None:
             remove_lead_trail_empty_lines=True,
             dedent=True,
         )
+
+
+# #############################################################################
+# TestDockerBuildProdImage1
+# #############################################################################
+
+
+class TestDockerBuildProdImage1(hunitest.TestCase):
```

Comment:
let's call it Test_docker_build_prod_image1 so it's easier to grep.
You can rename the others too.

We are ok to override PEP8 rules here.